src/dataflow/lib/iot.ts

```diff
@@ -11,6 +11,7 @@ const AWS_IOT_ENDPOINT_HOST = "a2zxjwmcl3eyqd-ats.iot.us-east-1.amazonaws.com";
 const OWNER_ID = "123";
 const SEND_INTERVAL = 1;
 const MAX_HUBS = 200;
+const HUB_RESPONSE_TIME = 60 * 1000;
```

Comment:
```suggestion
const HUB_RESPONSE_TIMEOUT = 60 * 1000;
```
Seems a bit clearer to me. Separately, 60 seconds seems like a long time to me, but your intuition is probably better than mine.

Reply:
This felt long to me too, but this is what the Dataflow team apparently requested.  I say we keep it at 60 seconds for now and see if they recommend a change once they get to play with it.

Reply:
Fixed name of contsant.